Cargo.toml

```
@@ -16,14 +16,14 @@ clap = "4.5"
 fnv = "1.0"
 futures = { version = "0.3", features = ["compat"] }
 pin-project = "1.1"
-product-config = { git = "https://github.com/stackabletech/product-config.git", tag = "0.6.0" }
+product-config = { git = "https://github.com/stackabletech/product-config.git", tag = "0.7.0" }
 rstest = "0.19"
 semver = "1.0"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 serde_yaml = "0.9"
 snafu = "0.8"
-stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "stackable-operator-0.67.1" }
+stackable-operator = { git = "https://github.com/stackabletech/operator-rs.git", tag = "stackable-operator-0.70.0" }
 strum = { version = "0.26", features = ["derive"] }
 tokio = { version = "1.37", features = ["full"] }
```

It's superficial since this sets the minimum version, but the there is a newer version we can set to a minimum:

```suggestion
tokio = { version = "1.38", features = ["full"] }
```

---

Also https://github.com/stackabletech/hello-world-operator/pull/104/commits/2966812089d7d4798718026100e0d5aa7dafb515